operations/observability/mixins/cross-teams/dashboards/gitpod-overview.json

```diff
@@ -100,7 +168,7 @@
             "uid": "$datasource"
           },
           "editorMode": "code",
-          "expr": "avg_over_time(sum(gitpod_ws_manager_mk2_workspace_phase_total{phase=\"Running\"})[7d])",
+          "expr": "avg_over_time(sum(gitpod_ws_manager_mk2_workspace_phase_total{phase=\"Running\",cell=\"$cell\"})[7d:10m])",
```

Comment:
In Gitpod Saas Grafana we got some result but a warnings of `Selected metric looks like a counter. Consider calculating rate of counter by adding rate().` and in Gitpod Dedicated Grafana the expression didn't work with an error of `bad_data: invalid parameter "query": 1:80: parse error: ranges only allowed for vector selectors`

The `[7d:10m]` turns the sum into a range vector by summing the metrics at intervals of 10m for the last 7 days. This makes Grafana happy in both cases. It is supposedly a bit expensive, and it is recommended that you use recording rules instead for things like this. Let starts with this and see how it goes.